### ISSUE TITLE
Blending hue_plus and grid-control

### ISSUE THREAD

**Issue by kusti8** [akej74/grid-control#12]
Hi,
I sent you a message on Reddit but I'm not sure if you got it. I'm interested in combining our drivers into one CAM replacement. I've already worked Leviathan into my code and would be interested to incorporate your code as well. The people on /r/buildapc really appreciated the work and it would be very useful to have a full replacement. I was wondering if you could provide me with info on how I could use your code and remake the UI into my own product or how we could combine them. From a glance, our code structures are a bit different in respect to pyqt5 usage. I'm sure I can figure it out, but I don't have a grid+ to test with.

**Comment by akej74** [comment]
Hi, just replied on Reddit before I saw your comment here. This is a good idea, let's continue the discussion.

**Comment by kusti8** [comment]
I am getting ready to release a beta version. The latest build is in
appveyor. Just a few bugs to iron out. I also modified your source so it
uses psutil if on Linux so it can get sensor data cross platform.

**Comment by akej74** [comment]
Good news, let's monitor the user comments and feedback on your [Reddit post](https://www.reddit.com/r/NZXT/comments/66rvfw/beta_testers_wanted_a_full_cam_alternative/)

**Comment by kusti8** [comment]
Yeah there was one person who was having troubles that I couldn't figure
out.

**Comment by themike10452** [comment]
I just finished my personal version of grid control, i added support for Hue+ and Kraken x52/x62. Since I only own 2 Hue Fans and no LED strips I cannot figure out how to control channel 2 for Hue+ so mine supports one channel currently. Kraken is fully working with libusb drivers. What happened to your project? Do we have an all-in-one alternative yet for CAM?

**Comment by akej74** [comment]
Hi, thanks for the update, it's great that you got Hue+ and Kraken to work; I have only done minor additions to Grid Control, like option to start minimized etc.

Not sure if there an all-in-one solution for CAM yet... Maybe we should start a new discussion about this on the NZXT subreddit, just to align where we are and get some idea of the different options?
